platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/jupiter/JUnitJupiterTestFrameworkIntegrationTest.groovy

```diff
@@ -82,11 +88,21 @@ class JUnitJupiterTestFrameworkIntegrationTest extends AbstractTestFrameworkInte
 
     @Override
     String getPassingTestCaseName() {
-        return "pass()"
+        return "pass"
     }
 
     @Override
     String getFailingTestCaseName() {
+        return "fail"
+    }
+
+    @Override
+    String getPassingTestMethodName() {
+        return "pass()"
```

Comment:
💭 It would be better to consolidate this logic in the VerifiesGenericTestReportResults or GenericHtmlTestExecutionResult (or a new reusable helper).

Reply:
This is consolidated in `TestFramework` now.